torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -401,14 +394,14 @@ void IRPrinter::visit(const For* v) {
   } else {
     os() << "{}";
   }
-  os() << std::endl;
```

Comment:
Where are these printed now? I don't see them getting printed in `Block`.

Reply:
The newlines?  A newline is printed at the end of each statement in the body of a `Block`.

Reply:
As I noted below, I don't see any newline at the end of each statement in a `Block`. Am I missing something?

Also, I see newlines at the end of some statements, like `Store`, `Let`, but not at the end of `For` and `Block`.

Reply:
Oh shoot.  I think I pushed an old version of this diff.  One moment.